content/docs/upgrading/upgrade.md

```diff
@@ -4,42 +4,44 @@ description = "Upgrading your deployment to a later version of Kubeflow"
 weight = 30
 +++
 
-Until version 1.0 Kubeflow makes no promises of backwards compatibility or 
+Until version 1.0, Kubeflow makes no promises of backwards compatibility or 
 upgradeability. Nonetheless, here are some instructions for updating your 
 deployments.
 
 ## Upgrading Kubeflow
 
-Updating your deployment is a two step process:
+Follow these steps to upgrade your Kubeflow deployment:
```

Comment:
@richardsliu @sarahmaddox KFupgrades won't be working with 0.7.0. Should we remove this section altogether or provide some sort of disclaimer?

Reply:
These instructions are not using the new upgrades logic. We can make the disclaimer above more explicit.

Also should we ask the users to delete their existing clusters first? I think directly applying the packages won't work.

Reply:
Thanks both. Updated. PTAL.